tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -61,10 +90,19 @@ def test_wazuh_db_messages(configure_sockets_environment, connect_to_sockets_mod
         List of test_case stages (dicts with input, output and stage keys).
     """
     for stage in test_case:
+        if 'ignore' in stage and stage['ignore'] == "yes":
+            continue
+        
         expected = stage['output']
         receiver_sockets[0].send(stage['input'], size=True)
         response = receiver_sockets[0].receive(size=True).decode()
-        assert response == expected, 'Failed test case stage {}: {}'.format(test_case.index(stage) + 1, stage['stage'])
+        
+        if 'use_regex' in stage and stage['use_regex'] == 'yes':
+            match = regex_match(expected, response)
+        else:
+            match = (expected == response)
+        assert match, 'Failed test case stage {}: {}. Expected: {}. Response: {}'\
+               .format(test_case.index(stage) + 1, stage['stage'], expected, response)
```

Comment:
If you need to know the index of `stage`, it is more efficient if you set the for like this:
```python
for index, stage in enumerate(test_case):
```

This way, you have the current index available in the variable `index`.

Reply:
Done.